r2/r2/controllers/ipn.py

```diff
@@ -590,9 +590,9 @@ def finish(self, parameters, txn_id,
                         "supports the site and makes future development "
                         "possible. For example, one month of reddit gold "
                         "pays for 5 instance hours of reddit's servers.")
-            message += "\n\n" + strings.gold_benefits_msg
+            message += "\n\n" + _(strings.gold_benefits_msg)
```

Comment:
Whoops. Out of curiosity can `_('hi %s bye' % data)` be properly translated?

If it can't be untranslated there's only two strings that would need to have both a translated and untranslated version (lounge_msg and gold_benefits_msg)